nixie/cli.py

```
@@ -159,11 +160,15 @@ async def _run_mermaid_cli(
 
     logging.getLogger(__name__).info(shlex.join(cmd))
 
+    allowed_executables = {"mmdc", "bun", "npx"}
+    if cmd[0] not in allowed_executables:
+        raise ValueError(f"Unexpected executable: {cmd[0]}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)